gui/tabs/players_tab.cpp

```diff
@@ -41,44 +41,42 @@ namespace PlayersTab {
 				if (State.selectedPlayerId > -1 && GetPlayerData(GetPlayerControlById(State.selectedPlayerId)) != nullptr)
 				{
 
-					if (IsInMultiplayerGame() && !IsHost()) {
-						if (!GetPlayerData(*Game::pLocalPlayer)->fields.IsDead) {
-							if (ImGui::Button("Call Meeting")) {
-								PlayerControl_CmdReportDeadBody(*Game::pLocalPlayer, NULL, NULL);
-							}
-							ImGui::NewLine();
-							if (State.selectedPlayerId > -1 && State.selectedPlayer != *Game::pLocalPlayer && ImGui::Button("Report Body")) {
-								PlayerControl_CmdReportDeadBody(*Game::pLocalPlayer, GetPlayerData(State.selectedPlayer), NULL);
-							}
+					if (!GetPlayerData(*Game::pLocalPlayer)->fields.IsDead) {
+						if (ImGui::Button("Call Meeting")) {
+							State.rpcQueue.push(new RpcReportPlayer(*Game::pLocalPlayer, NULL));
+						}
+						ImGui::NewLine();
+						if (State.selectedPlayerId > -1 && State.selectedPlayer != *Game::pLocalPlayer && ImGui::Button("Report Body")) {
+							State.rpcQueue.push(new RpcReportPlayer(*Game::pLocalPlayer, State.selectedPlayer));
 						}
+					}
 
-						if (State.selectedPlayerId > -1 && !GetPlayerData(State.selectedPlayer)->fields.Disconnected && State.selectedPlayer != *Game::pLocalPlayer)
+					if (State.selectedPlayerId > -1 && !GetPlayerData(State.selectedPlayer)->fields.Disconnected && State.selectedPlayer != *Game::pLocalPlayer)
```

Comment:
need a check to make sure we don't spectate a ghost while we are alive. something tells me that would look very odd to the server. i could be wrong though.

Reply:
> just recommending some small sanity checks. i know they aren't related to what you're actually changing but if we are pushing this through, might as well change them now anyways.
> 
> how sure are we that removing the muliplayer and host checks won't lead to people getting banned?

I've been playing for about a week now, test calling random meetings and reporting bodies.  Only bans I get is if I call a meeting or report as another player.  As long as you call from pLocalPlayer you're good.

Reply:
Please make a new PR with our rebased main branch and I'll test it.